Modules/galaxy/galaxy.py

```diff
@@ -55,6 +90,36 @@ def __init__(self, url: str = None):
                               name=sys['name'],
                               spectral_class=sys.get('spectral_class'))
 
+    async def find_nearest_landmark(self, system: StarSystem) -> Tuple[StarSystem, float]:
+        """
+        Find the nearest "landmark" system to the one provided. A list of landmark systems
+        can be found in Galaxy.LANDMARK_SYSTEMS.
+
+        Args:
+            system (StarSystem): The system to center the search around.
+
+        Returns:
+            A tuple containing the landmark StarSystem closest to the one provided, and a float
+            value indicating the distance between the two.
+        """
+
+        if system.name.casefold() in self.LANDMARK_SYSTEMS:
+            return (system, 0)
+
+        closest_system = None
+        closest_distance = -1
+        for _, landmark in self.LANDMARK_SYSTEMS.items():
+            distance = system.distance(landmark)
+            if closest_distance == -1 or distance < closest_distance:
+                closest_distance = round(distance, 2)
+                closest_system = landmark
+
+        if closest_system is None:
+            # Something's gone terribly wrong, likely the LANDMARK_SYSTEMS dict is empty.
+            raise RuntimeError(f"Could not find a closest landmark match for '{system.name}'!")
```

Comment:
methinks `ValueError` is more accurate here.

Reply:
I was debating on that fact for a while. The way I see it, the only way that this function could possibly fail, is if there are no `LANDMARK_SYSTEMS` defined within the class.

As I understand it, `ValueError` is meant to be used for when an argument is somehow invalid. But as long as it's a valid system, there should be no invalid values. Thus, having an empty landmark dict is a problem with the Galaxy class, not with the argument.

Reply:
Missed this reply, woops.

Your reasoning here is perfectly valid. 👍